packages/db/src/migrations/20250915152638_create_granule_groups_table_and_index.ts

```diff
@@ -0,0 +1,36 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  await knex.schema.createTable('granule_groups', (table) => {
```

Comment:
Should the name `granule_groups` be more specific, like `granule_duplicate_groups`, `granule_replacement_groups`?
How do we query the association of the granules within the same group?